pandas/core/generic.py

```diff
@@ -7538,16 +7538,21 @@ def between_time(
         Parameters
         ----------
         start_time : datetime.time or str
+                The first time value.
         end_time : datetime.time or str
+                The second time value.
         include_start : bool, default True
+                Adding start_time value in the result.
         include_end : bool, default True
+                Adding end_time value in the result.
         axis : {0 or 'index', 1 or 'columns'}, default 0
-
+                Determine range time on index or columns value.
             .. versionadded:: 0.24.0
 
         Returns
         -------
         Series or DataFrame
+            If axis set on columns, it will return DataFrame format, vice versa.
```

Comment:
I'd say something like "data from the original object filtered to the specified dates"